### ISSUE TITLE
Unable to run app ...

### ISSUE THREAD

**Issue by aiamuzz** [flathub/org.gnome.bijiben#4]
Hi,

Unable to run the app ... error below ...

`flatpak run org.gnome.bijiben `

```
** (bijiben:3): WARNING **: 21:26:13.953: Unable to connect to Tracker: Failed to load SPARQL backend: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Tracker1 was not provided by any .service files

** (bijiben:3): WARNING **: 21:26:13.953: Cannot initialize BijiManager: Failed to load SPARQL backend: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Tracker1 was not provided by any .service files
```

**Comment by TingPing** [comment]
Install tracker on the host. Ideally it could bundle what is needed or at least have a helpful error though.

**Comment by igaldino** [comment]
Hi @aiamuzz, thanks for your report. Yeah, this is something we need to work better and I hope to have something in the next releases. For now, please install Tracker. Thanks.

**Comment by igaldino** [comment]
I have opened an issue upstream: https://gitlab.gnome.org/GNOME/gnome-notes/issues/107

**Comment by igaldino** [comment]
Fixed upstream and ready to be deploy on 3.32 cycle.
